### DISCUSSION TITLE
StorageOperate

### DISCUSSION THREAD

**Discussion by Tiandmm** [apache/dolphinscheduler#13259] [General]
Task execute failed, due to meet an exception
org.apache.dolphinscheduler.common.exception.StorageOperateNoConfiguredException: Storage service config does not exist!

这里是哪个配置？

**Comment by Tiandmm** [comment]
我知道了，配置没有弄好

**Reply by xqhxxx** [reply]
您好，请问一下您，应该如何配置？。我已经按照官方文档配置为：resource.storage.type=HDFS
 ，resource.storage.upload.base.path，resource.hdfs.fs.defaultFS=这几个。然后提交spark任务的时候（可以上传资源jar包），但是执行就和您上面一样的报错。

**Reply by xujiangfeng** [reply]
您好，我也遇到同样的问题，我把所有的common.properties文件中的resource.storage.type属性都改成了HDFS，最后还是会报上面的错误，导致我可以上传文件到资源中心却无法引用资源中心中的文件，希望您有空能帮助解答。

**Reply by Scalaaaaaa** [reply]
我突然想起来了 @xqhxxx @xujiangfeng ,  我在上传jar的时候, 提示storage不存在,  于是就去api-server下的配置文件common.properties修改了存储的配置, 改成hdfs, 然后这个报错四运行时的,那么应该就是要改  worker-service下的common.properties配置

**Reply by Itsuka-Xx** [reply]
我也碰到这个问题，请问是怎么解决的？
